code/website/index.html

```diff
@@ -27,17 +34,28 @@ <h3 id="caseStudies">Case Studies</h3>
           <a href="$url$">[$type$]</a>
         $-endfor$
       </li>
-    $-if(src)$
-      <li><a href="$src-$">Generated Code</a></li>
-    $-endif$
+      $-if(src)$
+        <li>
+          Generated Code:
+          $-for(src)$
+            <a href="$path$">[$lang$]</a>
+          $-endfor$
+        </li>
+      $-endif$
     </ul>
     <br>
   </li>$sep-$
 $endfor$
 </ul>
+
 $if(docsUrl)-$
-<h4><a href="$docsUrl$">Haskell Documentation</a></h4>
+  <h3 id="haskDoc">Haskell Documentation</h3>
+  <p>
+    The current documentation for the Drasil framework can be found at
+    <a href="$docsUrl$">this link</a>.
```

Comment:
Suggestion: I'm not a fan of "this link" (even though I'm occasionally guilty of doing it); maybe something like "The (link)Haddock Documentation(/link) for the Drasil framework." @JacquesCarette or @smiths may have other suggestion or preferences though.

Reply:
I agree with @Mornix.  "this link" doesn't tell you what it is if you just scan the links.  I like the Haddock Documentation suggestion.

Reply:
Currently I have:
`The current <a href="$docsUrl$">Haddock documentation</a> for the Drasil framework.`

Reply:
Good - I completely agree that "this link" style is bad.